Components/Sources/Common/Views/SeparatorView.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  SeparatorView.swift
+//  
+//
+//  Created by Антон Голубейков on 23.05.2023.
+//
+
+import UIKit
+import ReactiveDataDisplayManager
+
+/// Base view to implement separator between other views or cells.
+public class SeparatorView: UIView {
+
+    // MARK: - Private properties
+
+    private var heightConstraint: NSLayoutConstraint?
+
+    // MARK: - Initialization
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureConstraints()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+}
+
+// MARK: - ConfigurableItem
+
+extension SeparatorView: ConfigurableItem {
+
+    public struct Model {
```

Comment:
```suggestion
    public struct Model: Equatable {
```
суть в том чтобы все ячейки из компонентов подготовить к использованию с diffableDataSource.
А для этого достаточно привести модельку к Equatable.